libs/ledger-live-common/package.json

```diff
@@ -240,6 +242,7 @@
     "tsx": "^4.7.1",
     "usehooks-ts": "^2.13.0",
     "utility-types": "^3.10.0",
+    "web3": "^4.7.0",
```

Comment:
we use 
the "ethers" lib instead of "web3" in our codebase.

i.e.
https://github.com/LedgerHQ/ledger-live/blob/develop/libs/coin-modules/coin-evm/package.json#L75

could you plz change it to "ethers" lib to be coherent with the rest of our codebase? Thanks a lot

Reply:
Definitely, now using ethers instead of web3.